app/models.py

```diff
@@ -493,6 +513,14 @@ class User(Base, ModelMixin, UserMixin, PasswordOracle):
         nullable=False,
     )
 
+    # Keep original unsub behaviour
+    unsub_behaviour = sa.Column(
```

Comment:
should we have a migration to convert `one_click_unsubscribe_block_sender` to this new column?

Reply:
I was thinking of  running a migration script just after this is deployed.

Reply:
I think we can add a sql query into the migration file, something like https://github.com/simple-login/app/blob/02a005d07647fa75b4f5271a4e56f8f1ce412fd7/migrations/versions/2021_102715_0b9150eb309d_.py#L22

```
op.execute("update users set unsub_behaviour=0 where one_click_unsubscribe_block_sender=false;")
op.execute("update users set unsub_behaviour=1 where one_click_unsubscribe_block_sender=true;")

```

Reply:
Ah. I though you didn't want non auto-generated alembic ops in the migrations. That's how we wanted to do it for proton partner and did a manual migration. Will add this to the migration :+1: